hack/local-up-karmada.sh

```diff
@@ -131,7 +131,10 @@ util::create_cluster "${PULL_MODE_CLUSTER_NAME}" "${PULL_MODE_CLUSTER_TMP_CONFIG
 #step2. make images and get karmadactl
 export VERSION="latest"
 export REGISTRY="docker.io/karmada"
+export LABEL="repository=karmada.io"
```

Comment:
```suggestion
export KARMADA_IMAGE_LABEL="repository=karmada.io"
```
Adding karmada to the variable's name may avoid conflicts with existing variables on the environment.

Reply:
SGTM, done.